README.md

```diff
@@ -6,3 +6,8 @@
 [![License](https://img.shields.io/github/license/open-cluster-management/hub-of-hubs-data-types)](/LICENSE)
 
 The data-types module of [Hub-of-Hubs](https://github.com/open-cluster-management/hub-of-hubs).
+
+## DeepCopy and CRD generation
+
+1. DeepCopy - run ```make generate```. The file will be generated in the corresponding API folder
+2. CRD - run ```make manifests```. The YAML files will be created under <root>/crd folder
```

Comment:
Could you please add a sentence that after generating the CRDs, they must be copied into the APIs repo?

Reply:
done